src/extension/ncwms/src/main/java/au/org/emii/geoserver/wms/NcwmsConfig.java

```diff
@@ -68,4 +69,27 @@ public String getConfigVariable(String xpathString, String defaultValue) {
 
         return returnValue;
     }
+
+    public List<String> getConfigList(String xpathString) {
+        SAXReader reader = new SAXReader();
+
+        List<String> returnValue = new ArrayList<>();
+
+        try {
+            Document doc = reader.read(getConfigFile());
+            DefaultXPath xpath = new DefaultXPath(xpathString);
+
+            @SuppressWarnings("unchecked")
+            List<DefaultElement> list = xpath.selectNodes(doc);
+
+            for (final DefaultElement element : list) {
+                returnValue.add(element.getText());
+            }
+        }
+        catch (DocumentException e) {
+            LOGGER.log(Level.WARNING, String.format("Could not open config file '%s': '%s'", getConfigFile(), e.getMessage()));
```

Comment:
Is it possible to have DocumentException's that are something other than failing to open the config file?